cmd/mistryd/job_test.go

```diff
@@ -65,4 +65,15 @@ func TestJobID(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+
+	ignParams := params
+	ignParams["_production"] = "ignored"
```

Comment:
Better use a consistent glossary. Let's call those opqParams (for "opaque").

Reply:
Done